drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -14,11 +14,9 @@ class Enrollment < ::HudReports::ReportClientBase
     include HudReports::Clients
     acts_as_paranoid
 
-    # These should probably live somewhere else, but for now, they live here
-    # they come from the HUD DQ report Q5
-    HOMELESS_LIVING_SITUATIONS = [16, 1, 18].freeze
-    INSTITUTIONAL_LIVING_SITUATIONS = [15, 6, 7, 25, 4, 5].freeze
-    HOUSED_LIVING_SITUATIONS = [29, 14, 2, 32, 36, 35, 28, 19, 3, 31, 33, 34, 10, 20, 21, 11, 8, 9, 99].freeze
+    HOMELESS_LIVING_SITUATIONS = HudUtility2024::SITUATION_HOMELESS_RANGE.to_a
```

Comment:
It looks like this is only used for Prior Living Situations, so it would probably be better to use `HudUtility2024.homeless_situations(as: :prior)`. That gives us the actual valid homeless situations, instead of the whole range of 100 numbers, most of which are invalid.

Though it would depend if you want to include situations that are potentially invalid but that we can still categorize -- for example if a Destination situation is stored as the answer to PLS. It's invalid technically, but we still can figure out which category it's in.

Reply:
Good call, I'll make that change.